aw_analysis/query2.py

```diff
@@ -5,15 +5,12 @@
 from aw_core.models import Event
 from aw_datastore import Datastore
 
+from .query2_error import QueryParseException, QueryInterpretException
 from .query2_functions import query2_functions
 
 logger = logging.getLogger(__name__)
 
 
-class QueryException(Exception):
```

Comment:
Removing this led to some tests failing in CI, better to mark it deprecated.

Reply:
Fixed, I moved it to query2_error.py so by simply importing it in this file fixed it.